src/ui/atoms/logo/Logo.tsx

```diff
@@ -0,0 +1,57 @@
+import classNames from 'classnames'
+import React, { useCallback } from 'react'
+import type { ThemeContextType } from 'context/themeContext'
+import { useTheme } from 'hook/useTheme'
+import DarkLogo from '../../../assets/logos/logo-okp4-dark.svg'
+import LightLogo from '../../../assets/logos/logo-okp4-light.svg'
+import LightSlogan from '../../../assets/logos/slogan-okp4-light.svg'
+import DarkSlogan from '../../../assets/logos/slogan-okp4-dark.svg'
+import LightLogotype from '../../../assets/logos/logotype-okp4-light.svg'
+import DarkLogotype from '../../../assets/logos/logotype-okp4-dark.svg'
+import LightLogomark from '../../../assets/logos/logomark-okp4-light.svg'
+import DarkLogomark from '../../../assets/logos/logomark-okp4-dark.svg'
+
+import './logo.scss'
+
+export type TLogoProps = Readonly<{
+  /**
+   * The size of the logo, must be determined according to its use and location.
+   */
+  readonly size?: 'small' | 'medium' | 'large'
+  /**
+   * The type of the logo, flexible according to use and location.
+   * By default, the standalone logo including the logotype, the logomark and the slogan..
+   */
+  readonly type?: 'logo' | 'logotype' | 'logomark' | 'slogan'
+}>
+
+export const Logo: React.FC<TLogoProps> = ({ size = 'medium', type = 'logo' }: TLogoProps) => {
+  const { theme }: ThemeContextType = useTheme()
+
+  const getElement = useCallback(() => {
```

Comment:
Why not` useMemo()` since you want to memorize a value rather than a function I think? But from what I understand, I'm not sure it's useful since the calculation performed is not really expensive, isn't it?

Reply:
@fredericvilcot at your disposal to discuss this point if you wish, there are certainly things I might be missing.

Reply:
@ccamel As we discussed, the convention here is to avoid a new function to be created when component proceed a new render. 
I propose to use `useCallback` for all the functions / methods created in a React component in order to benefit from memoization mechanisms and to prevent reliance on a global order function.

Reply:
Good. Thx for explanations.